docroot/themes/humsci/humsci_basic/src/scss/utilities/_lists.scss

```diff
@@ -76,7 +76,9 @@
 
     // If a link is an external icon, make space for it.
     .fa-ext {
-      padding-right: hb-calculate-rems(15px);
+      &::after {
+        background-color: transparent;
+      }
```

Comment:
We may be able to remove this if we find another solution for the background-color of the icon.